core/src/kmx/kmx_file.h

```diff
@@ -340,11 +340,18 @@ struct COMP_KEYBOARD {
   KMX_DWORD dwBitmapSize;   // 003C size in bytes of the bitmaps
 };
 
-// TODO-LDML: fill this in
-struct COMP_KEYBOARD_EX {
+struct COMP_KEYBOARD_KMXPLUSINFO {
   KMX_DWORD dpKMXPlus;      // 0040 offset of KMXPlus data, <sect> header is first
-  KMX_DWORD dwKMXPlusSize;  // 0048 size in bytes of entire KMXPlus data
-  };
+  KMX_DWORD dwKMXPlusSize;  // 0044 size in bytes of entire KMXPlus data
+};
+
+/**
+ * Only valid if comp_keyboard.dwFlags&KF_KMXPLUS
+ */
+struct COMP_KEYBOARD_EX {
+  COMP_KEYBOARD comp_keyboard;              // 0000 see COMP_KEYBOARD
+  COMP_KEYBOARD_KMXPLUSINFO comp_keyboard_kmxplusinfo;  // 0040 see COMP_KEYBOARD_EXTRA
```

Comment:
```suggestion
  COMP_KEYBOARD             header;    // 0000 see COMP_KEYBOARD
  COMP_KEYBOARD_KMXPLUSINFO kmxplus;   // 0040 see COMP_KEYBOARD_EXTRA
```
I wonder if we can just keep them a bit shorter?